setup.py

```diff
@@ -46,4 +67,11 @@
         "Programming Language :: Python :: Implementation :: CPython",
         "Programming Language :: Python :: Implementation :: PyPy",
     ],
+    keywords=" ".join([
+        "math",
+        "probability",
+        "numerical",
+        "symbolic",
+        "MCMC"
+    ])
```

Comment:
I think these can be provided as a plain list without the concatenation.

Reply:
Indeed: https://setuptools.pypa.io/en/latest/references/keywords.html